Makefile

```
@@ -58,6 +59,14 @@ assemble:
 buildPlugins:
 	./gradlew copyPluginZip
 
+#
+# Build plugins and copy to the local Nextflow plugins directory
+#
+install:
+	./gradlew copyPluginZip
+	rm -rf ${HOME}/.nextflow/plugins/nf-schema-${version}
```

Should this be 

```suggestion
	rm -rf ${HOME}/.nextflow/plugins/nf-lamin-${version}
```

?